platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/buses/I2CGenerator.xtend

```diff
@@ -103,9 +123,66 @@ class I2CGenerator extends AbstractSystemResourceGenerator {
 		}
 	}
 	
+	private def Boolean inverseByteOrder(SignalInstance sigInst) {
```

Comment:
this function sounds like it's actually inverting a byte order when it's actually determining wether the byte order should be inverted. Please rename to something like `shouldInvertByteOrder` or `isInverseByteOrder`